### PR TITLE
[AUD-1011] Add connect wallets drawer

### PR DESCRIPTION
**Pull request by Kyle-Shanks**
### Description
Add connect wallets drawer

### How Has This Been Tested?

Manually tested. It looks great

### CODE REVIEW

File: src/components/connect-wallets-drawer/ConnectWalletsDrawer.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    text: {
+      fontFamily: 'AvenirNextLTPro-Regular',
+      textAlign: 'center',
+      fontWeight: '500',
+      fontSize: 24,
+      lineHeight: 30
+    }
+  })
+
+// `ActionDrawer` is a drawer that presents a list of clickable rows with text
```
Comment (sliptype) [inline src/components/connect-wallets-drawer/ConnectWalletsDrawer.tsx:52]: Extra comment

File: src/components/connect-wallets-drawer/ConnectWalletsDrawer.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  StyleSheet.create({
+    container: {
+      paddingTop: 48,
+      paddingLeft: 16,
+      paddingRight: 16,
+      paddingBottom: 48
+    },
+
+    title: {
+      fontFamily: 'AvenirNextLTPro-Heavy',
```
Comment (sliptype) [inline src/components/connect-wallets-drawer/ConnectWalletsDrawer.tsx:36]: No need to change but you can set `weight="heavy"` on our `Text` component. I think it's a little cleaner than typing out the font family everywhere

**Review by sliptype — APPROVED** [review]
Nice!

File: src/components/connect-wallets-drawer/ConnectWalletsDrawer.tsx (hunk 2 above)
Comment (Kyle-Shanks) [reply src/components/connect-wallets-drawer/ConnectWalletsDrawer.tsx]: I agree. Used font weight so I don't have to write it twice though 👍

Comment (sliptype) [reply src/components/connect-wallets-drawer/ConnectWalletsDrawer.tsx]: Font-weight doesn't work on Android. Or iOS? one of the two doesn't work lol

Also I think eventually we should turn the masked text thing into a component so we don't have all this duplication everywhere

**Pull request merged**